ProteoformSuiteGUI/AggregatedProteoforms.cs

```diff
@@ -42,12 +42,15 @@ private void run_the_gamut()
             this.Cursor = Cursors.WaitCursor;
             Lollipop.aggregate_proteoforms(Lollipop.validate_proteoforms, Lollipop.raw_neucode_pairs, Lollipop.raw_experimental_components, Lollipop.raw_quantification_components, Lollipop.min_rel_abundance, Lollipop.min_num_CS);
             FillAggregatesTable();
-            ((ProteoformSweet)MdiParent).experimentalTheoreticalComparison.ClearListsAndTables();
-            ((ProteoformSweet)MdiParent).experimentalTheoreticalComparison.run_the_gamut();
-            ((ProteoformSweet)MdiParent).experimentExperimentComparison.ClearListsAndTables();
-            ((ProteoformSweet)MdiParent).experimentExperimentComparison.run_the_gamut();
-            ((ProteoformSweet)MdiParent).quantification.ClearListsAndTables();
-            ((ProteoformSweet)MdiParent).quantification.perform_calculations();
+            if (Lollipop.proteoform_community.theoretical_proteoforms.Length > 0)
```

Comment:
Let's clear the tables every time, and then use the neucode_labeled condition, since running through this all is useful for NeuCode, which requires less manual input. `if (Lollipop.neucode_labeled && Lollipop.proteoform_community.theoretical_proteoforms.Length > 0)`